### DISCUSSION TITLE
EmulatorJS on an AndroidTV?

### DISCUSSION THREAD

**Discussion by kmblaszak** [EmulatorJS/EmulatorJS#844] [General]
Hello all, 
           I have an EmulatorJS server running on an ubuntu home server and it runs games perfectly when I play them on my home PC via chrome browser or my phone but I wanted to see if there was a way to get these games on my android TV as well? Has anyone tried this and had any success? It seems like a lot of the browsers are either non-chromium based or lack sufficient controller support to work well on a TV. 

I've loaded Chrome on the TV and tried with an Xbox controller with no luck. Ironically the TV remote seems to work well but without the ability to add extensions I think I might be a little stuck. Anyone have any ideas?

**Comment by Underdog-01** [comment]
For what you currently have going but are having trouble with game controllers & assuming the EmulatorJS built in keymapper is not detecting your game controller buttons (accessed via the the control icon on the EmulatorJS index) , you might need some sort of 3rd party key mapper to make the controllers buttons map to existing keyboard keys (ie. A, X, D, ENT, SPACE, etc.). This will end up mapping the keys twice though.. seems a bit odd but whatever it takes.
